rgbd/src/server_rgbd.cpp

```diff
@@ -62,8 +62,8 @@ void ServerRGBD::send(const Image& image)
     if (pub_image_.getNumSubscribers() == 0)
         return;
 
-    rgbd_msgs::RGBD msg;
-    msg.version = MESSAGE_VERSION;
+    rgbd_msgs::RGBDPtr msg(new rgbd_msgs::RGBD);
```

Comment:
Here as well. Don't need to initialize a `new` message because `RGBDPtr` by default will be a sharedptr

Reply:
But then it will be a nullptr. So this is correct IMO

Reply:
Then use make_shared instead

Reply:
```suggestion
    rgbd_msgs::RGBDPtr msg = std::make_shared<rgbd_msgs::RGBD>();
```